### ISSUE TITLE
pmempool sync: it is possible to add secondary replica with smaller size than size of poolset

### ISSUE THREAD

**Issue by kamdiedrich** [pmem/issues#304]
Consider following local pool set file:

```
PMEMPOOLSET
20M /tmp/part.0
20M /tmp/part.1
```

Then we change this file:

```
PMEMPOOLSET
20M /tmp/part.0
20M /tmp/part.1
REPLICA
20M /tmp/partReplica.0
10M /tmp/partReplica.1
```

Sync will create this replica and after run pmempool check, exit code is 0. Which means that pool is consistent. From transform spec:
`Also, to add a replica it is necessary for its effective size to match or exceed the pool size`
From this example we loss the 'physical' size of a poolset.

Found on: 1.1-815-ga8bc37d

**Comment by plebioda** [comment]
@kamdiedrich  Could you please verify this issue ?

**Comment by kamdiedrich** [comment]
I was able to reproduce it on version: 1.1-1042-g260c6b2

**Comment by plebioda** [comment]
This issue has been fixed in 1.1-1043-gb39acd4

**Comment by kamdiedrich** [comment]
Verified on: 1.1-1092-gdbefada
